core/client/fs/src/main/java/alluxio/client/block/RetryHandlingBlockMasterClient.java

```diff
@@ -71,29 +74,31 @@ protected long getServiceVersion() {
 
   @Override
   protected void afterConnect() {
-    mClient = new BlockMasterClientService.Client(mProtocol);
+    //mClient = new BlockMasterClientService.Client(mProtocol);
+    mGrpcClient = BlockMasterClientServiceGrpc.newBlockingStub(mChannel);
```

Comment:
Channels are expensive according to gRPC community. However in production we won't have more than 2-3 clients per node. So I don't think it would affect the performance much.